### ISSUE TITLE
magic shovel

### ISSUE THREAD

**Issue by kentar91** [uomars/UOMars#1070]
Le magic shovel non danno bonus in quanto non possono essere impugnate.
Le facciamo impugnare?

**Comment by giga487** [comment]
No, manca la grafica della pala. O mi trovi un altro sistema simile la cui grafica esiste, o mi trovi una grafica di una pala online
